python/src/scipp/plot/sciplot.py

```diff
@@ -216,19 +223,15 @@ def _process_axes_dimensions(self,
             self.axes[key_list[ind]] = self.axes[key]
             self.axes[key] = dim
 
-        self._validate_axes(array)
-
-    def _validate_axes(self, array):
+    def validate(self):
         """
-        Validation checks on axes.
+        Validation checks before plotting.
         """
-        for dim, coord in array.coords.items():
-            if len(coord.dims) > 1:
-                self.multid_coord = dim
+        multid_coord = self.model.get_multid_coord()
```

Comment:
You also have a `self.multid_coord` variable, which I think you never use? you could avoid the temporary and just make the function call as part of the test.